.github/workflows/publish-draft-release.yml

```diff
@@ -65,49 +54,52 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ matrix.runtime }}-runtime
-          path: "${{ steps.build-runtime.outputs.wasm }}"
+          path: |
+            ${{ steps.srtool_build.outputs.wasm }}
+            ${{ steps.srtool_build.outputs.wasm_compressed }}
```

Comment:
Just curious, what happens when you use two lines for the path here?

Reply:
that are the names of two artifacts from `id: srtool_build` job

Reply:
The artifact `${{ matrix.runtime }}-runtime` is an id. Listing several files or a pattern means you end up with several files in this artifact.